kernel/pom.xml

```diff
@@ -532,20 +483,6 @@
           <warSourceDirectory>${basedir}/src/main/webapp</warSourceDirectory>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Well currently it is in the master pom protected under the sakai-release profile but that profile does a lot more (and version needs to be updated.)

It should be added to master if we want it overall added.

The real problem here is the gpg runs in the verify phase and the default for the source plugin was package phase, so it just needs to run before the gpg. So I can either move this plugin to master or leave it here and just remove the phase.